lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -729,7 +736,23 @@ void PatchInOutImportExport::visitCallInst(CallInst &callInst) {
           loc = resUsage->inOutUsage.outputLocMap[outLocInfo.u32All];
         }
       } else {
-        if (resUsage->inOutUsage.outputLocMap.find(value) != resUsage->inOutUsage.outputLocMap.end()) {
+        if (m_pipelineState->isPackInOut() && m_shaderStage == ShaderStageVertex &&
+            m_pipelineState->hasShaderStage(ShaderStageTessControl)) {
+          // The new InOutLocationInfo is used to map scalarized VS output export in a VS-TCS-TES-FS to lds as compact
+          // as possible
+          InOutLocationInfo origLocInfo = {};
+          origLocInfo.location = value;
+          origLocInfo.component = cast<ConstantInt>(callInst.getOperand(1))->getZExtValue();
+          origLocInfo.half = false;
+          if (resUsage->inOutUsage.inOutLocMap.find(origLocInfo.u16All) != resUsage->inOutUsage.inOutLocMap.end()) {
```

Comment:
use `llvm::is_contained `

Reply:
inOutLocMap is a std::map. llvm::is_contained seems not work on std::map::find.

Reply:
Ah, you are right. llvm::is_contained should be fixed to used to use the find method if available IMO, but that's completely out of scope of this revision